src-ghc/Pact/Server/ApiServer.hs

```diff
@@ -89,7 +89,7 @@ sendBatch public = do
   when (null cmds) $ die "Empty Batch"
   crs <- forM cmds $ \c -> do
     cr@(_,Command{..}) <- buildCmdRpc c
-    liftIO $ print cr
+    --liftIO $ print cr
```

Comment:
I commented this out because it was outputting to stdout even when verbose logging was turned off. If this is needed, I can undo this.